app/javascript/crayons/MentionAutocompleteTextArea/__stories__/MentionAutocompleteTextArea.stories.jsx

```diff
@@ -0,0 +1,99 @@
+import { h, createRef, render } from 'preact';
+import { MentionAutocompleteTextArea } from '../MentionAutocompleteTextArea';
+import notes from './mention-autocomplete.md';
+
+export default {
+  title: 'Components/MentionAutocompleteTextArea',
```

Comment:
Documentation (blocking): This seems to be more of an app component than component. If we rework the autocomplete to be generic for other use cases, that generic autocomplete would make sense in components.

```suggestion
  title: 'App Components/MentionAutocompleteTextArea',
```